src/pudl/settings.py

```diff
@@ -201,7 +201,7 @@ class Eia860Settings(GenericDatasetSettings):
     eia860m_date: ClassVar[str] = eia860m_data_source.working_partitions["year_month"]
 
     years: list[int] = data_source.working_partitions["years"]
-    eia860m: bool = True
+    eia860m: bool = False  # True # temp change! need to add 2023 monthly data
```

Comment:
Do we need to change the default value for the class, which we ought to be overriding in the ETL settings already, temporarily? It seems like in the general case we do still want to default to True -- and override it as necessary (as we're doing now).

Reply:
Yeah I agree. I wasn't sure the best way to make this a temporary fix that would have to be changed later once we add in monthly data. I suppose we could write something that would check for monthly data that was greater than the annual data available and if that was not true then eia860m bool would be set to false?

Reply:
Alternatively, I could just go ahead and integrate the 2023 monthly EIA 860 data

Reply:
I don't understand. Can't you just leave the fault True? The temporary changes you've made to the settings should override that default.

Reply:
I think this is a @bendnorman question.

Reply:
The `etl_fast` job pulls from `src/pudl/package_data/settings/elt_fast.yaml`, and `etl_full` uses the default values of the settings objects. We should probably create `etl_full` from the settings file instead so it's less confusing and we don't run into the issue where people are temporarily changing the default values to work with `etl_full`.

Want me to make a PR off of this branch?